lib/processor.js

```diff
@@ -85,6 +86,12 @@ Processor.prototype.getCollection = function getCollection(
   return Processor._mocks[databaseName][collectionName] || [];
 };
 
+Processor.prototype.collectionExists = function collectionExists(
+  databaseName,
+  collectionName) {
```

Comment:
Agree. I have increased indent here for the parameter list in order to visually separate the function parameters and the body.